_ml-commons-plugin/api/model-apis/update-model.md

```diff
@@ -130,3 +130,53 @@ PUT /_plugins/_ml/models/9uGdCJABjaMXYrp14YRj
+    }
+}
+```
+{% include copy-curl.html %}
+
+If the model was registered using the [Amazon Bedrock AI21 Labs Jurassic blueprint](https://github.com/opensearch-project/ml-commons/blob/2.x/docs/remote_inference_blueprints/bedrock_connector_ai21labs_jurassic_blueprint.md), a default interface is applied automatically.
+{: .note}
+
+If the model interface is no longer needed, you can remove both the input and output schemas in order to bypass model schema validation:
+
+```json
+PUT /_plugins/_ml/models/IMcNB5UB7judm8f45nXo
+{
+  "interface": {
```

Comment:
When we register model we can use `"interface": {}` to set empty interface. Why can't we use same way when update model ?
```
POST _plugins/_ml/models/_register
{
  "name": "...",
  "function_name": "remote",
  "description": "test model",
  "connector_id": "...",
  "interface": {}
}
```

Reply:
I link the model interface link from model registration in the above comment. Since this page is for update model, I think we better not circle back to register API. We should use the update model API as an example. Also, this is an example to show how to remove interface when it's no longer needed. I think the model is assumed registered already.